src/server_manager/web_app/ui_components/outline-manual-server-entry.js

```diff
@@ -188,6 +202,10 @@ Polymer({
               <iron-icon id="gcpCreateServerDropDownIcon" icon="arrow-drop-down"></iron-icon>
             </div>
           </div>
+          <div id="gcp-new-flow-promo" class="section-content-instructions" on-tap="gcpNewFlowTapped">
+            <div id="gcp-tag">[[localize('experimental')]]</div>
+            <a>[[localize('setup-gcp-promo')]]<iron-icon icon=open-in-new></iron-icon></a>
```

Comment:
The "open in new window" icon is not appropriate, since we open in the same window. We should remove it.

Instead, convert the link to text and add a button with the set up server call to action (same we use for DO)

Reply:
Clicking this link instantly opens the Google Oauth flow in a new window, so the icon seems natural to me.